src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/store/index.ts

```diff
@@ -1,36 +1,20 @@
-import {
-  applyMiddleware,
-  compose,
-  createStore,
-  Middleware,
-  Store,
-} from 'redux';
-import { composeWithDevTools } from 'redux-devtools-extension';
```

Comment:
`composeWithDevTools` was removed - I guess we dont really need this? If not, it can also be removed from package.json